setup.py

```diff
@@ -3,13 +3,45 @@
 # (c) 2018 Mario Kleiner - Licensed under MIT license.
 #
 
+# To build Python extensions/wheels, you'll want pip >= 10.0 for pyproject.toml support.
+# This file will allow pip to automatically download the required build dependencies (i.e. numpy).
+# To build this wheel locally into the dist/ directory, you could use the following
+# (assuming the current working directory is next to the setup.py file):
+#
+# > pip wheel . -w dist
+#
+# (see https://pip.pypa.io/en/stable/cli/pip_wheel/ for all options)
+#
+# To build and install in the same step:
+#
+# > pip install .
+#
+# (see https://pip.pypa.io/en/stable/cli/pip_install/ for all options)
+#
+# If you wish to build wheels for distribution, you may want to compile with Python's Limited API
+# enabled. The resulting wheel can be used across multiple Python releases (see
+# https://docs.python.org/3/c-api/stable.html for details).
+# This requires Python >= 3.7, and PTB_LIMITED_WHEEL must be set to 1. For example,
+#
+# > PTB_LIMITED_WHEEL=1 pip wheel . -w dist
+#
+# Will produce a wheel that will work for Python versions >= 3.7.
+# It is recommended to compile these on the oldest supported Python version (i.e. 3.7).
+
 # from distutils.core import setup, Extension # Build system.
 from setuptools import setup, Extension, find_packages
 import os, fnmatch, shutil                  # Directory traversal, file list building.
 import platform                             # OS detection.
 import sys                                  # cpu arch detection.
 import numpy                                # To get include dir on macOS.
 
+from wheel.bdist_wheel import bdist_wheel
+
+# We borrowed the custom wheel configuration from https://github.com/joerick/python-abi3-package-sample
+# It lives in a separate file for licensing simplicity
+sys.path.append(os.path.join(os.path.dirname(__file__), 'PsychPython', 'psychtoolbox'))
```

Comment:
Could also just put the _abi3_wheel.py next to setup.py for simplicity? I'm fine with either way.